at-hoangtran/Week3/Day4/Exercise2-Day4.rb

```diff
@@ -0,0 +1,12 @@
+class SumArray
+  def self.sum_arr(arr)
+    check = arr.all? { |x| x.is_a? Integer }
+    if check == true
+      arr.sum
+    else
+      nil
+    end
+  end
+end
+arr = [1, 2, 3, 4]
+p SumArray.sum_arr(arr)
```

Comment:
Sửa `p SumArray.sum_arr(arr)` thành `p [1,2,3,4].sum` và in ra kết quả đúng là 10